packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -191,16 +191,14 @@ function DataViews< Item >( {
 				renderItemLink,
 				containerWidth,
 				containerRef,
+				resizeObserverRef,
 				defaultLayouts,
 				filters,
 				isShowingFilter,
 				setIsShowingFilter,
 			} }
 		>
-			<div
-				className="dataviews-wrapper"
-				ref={ useMergeRefs( [ containerRef, resizeObserverRef ] ) }
-			>
+			<div className="dataviews-wrapper" ref={ containerRef }>
```

Comment:
I believe that `containerRef` needs to be on this wrapper because [its use](https://github.com/WordPress/gutenberg/blob/trunk/packages/dataviews/src/dataviews-layouts/table/index.tsx#L278) is related to the scrolling behaviour of the wrapper. 

The fact that this is the scrollable container for the layout is what makes it unsuitable to use with the resizeObserver, because the presence of a scrollbar will throw off the size of that actual layout. This is why I'm moving resizeObserverRef to context so it can be used directly in the grid layout.